src/Quasar/FS/Mount.purs

```diff
@@ -44,43 +45,70 @@ data MountF f
 type Mount = MountF Identity
 type MountType = MountF (Const Unit)
 
-
-instance eqMount ∷ (Eq1 f) ⇒ Eq (MountF f) where
-  eq =  case _, _ of
-    View x, View y -> eq1 x y
-    View x, _ -> false
-    Module x, Module y -> eq1 x y
-    Module _, _ -> false
-    MongoDB x, MongoDB y -> eq1 x y
-    MongoDB _, _ -> false
-    Couchbase x, Couchbase y -> eq1 x y
-    Couchbase _, _ -> false
-    MarkLogic x, MarkLogic y -> eq1 x y
-    MarkLogic _, _ -> false
-    SparkHDFS x, SparkHDFS y -> eq1 x y
-    SparkHDFS _, _ -> false
-    SparkLocal x, SparkLocal y -> eq1 x y
-    SparkLocal _, _ -> false
-    Mimir x, Mimir y -> eq1 x y
-    Mimir _, _ -> false
-    Unknown xName x, Unknown yName y -> eq1 x y && eq xName yName
-    Unknown _ _, _ -> false
+instance eqMount ∷ Eq1 f ⇒ Eq (MountF f) where
+  eq = case _, _ of
+    View x, View y → eq1 x y
+    View x, _ → false
+    Module x, Module y → eq1 x y
+    Module _, _ → false
+    MongoDB x, MongoDB y → eq1 x y
+    MongoDB _, _ → false
+    Couchbase x, Couchbase y → eq1 x y
+    Couchbase _, _ → false
+    MarkLogic x, MarkLogic y → eq1 x y
+    MarkLogic _, _ → false
+    SparkHDFS x, SparkHDFS y → eq1 x y
+    SparkHDFS _, _ → false
+    SparkLocal x, SparkLocal y → eq1 x y
+    SparkLocal _, _ → false
+    Mimir x, Mimir y → eq1 x y
+    Mimir _, _ → false
+    Unknown xName x, Unknown yName y → eq xName yName && eq1 x y
+    Unknown _ _, _ → false
```

Comment:
I just unicodeified this. Also swapped the `eq` part for `Unknown`, as string comparison on the names should be faster than `eq1` on the inner values, and `&&` short circuits.